subsys/bluetooth/host/adv.c

```diff
@@ -392,7 +392,7 @@ static bool valid_adv_ext_param(const struct bt_le_adv_param *param)
 	    BT_DEV_FEAT_LE_EXT_ADV(bt_dev.le.features)) {
 		if (param->peer &&
 		    !(param->options & BT_LE_ADV_OPT_EXT_ADV) &&
-		    !(param->options & BT_LE_ADV_OPT_CONNECTABLE)) {
+		    !(param->options & _BT_LE_ADV_OPT_CONNECTABLE)) {
```

Comment:
Should we reject all API calls with the internal bits set?

Reply:
Not possible the way I define the values. `_BT_LE_ADV_OPT_CONNECTABLE` has the same value as `BT_LE_ADV_OPT_CONNECTABLE`.

Reply:
Ah, right. Not much we can do there for now then I guess